users/templates/base.html

```diff
@@ -76,6 +78,7 @@
             {% if user.user_type == manager or user.user_type == admin %}
             <li>
                 {% if request.path == custom_projects_list_url %}
+                <!--highlight won't work now because of date parameters-->
```

Comment:
I don't know what to do about this. In order to have a tab highlighted, a full URL is checked. But the exact parameters are available in sub-template, so we cannot access them from here. Perhaps there'd be some way to reliably check for a part of URL? Or maybe something like a bool value that would be passed from the rendered view?

Anyway, should I solve this bug with this PR or just leave it for now and add an issue for it, so we can deal with it later?

Reply:
Aren't the available in `request` object which should be accessible in this template too?

Reply:
There were more same problems as it in SheetStorm. I fixed them added template_tag.